cpp/meta_module/algorithm/meta.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <mgp.hpp>
+#include <string_view>
+
+namespace Meta {
+
+/* update constants */
+constexpr std::string_view kProcedureUpdate = "update";
+constexpr std::string_view kUpdateArg1 = "createdObjects";
+constexpr std::string_view kUpdateArg2 = "deletedObjects";
+constexpr std::string_view kUpdateArg3 = "removedVertexProperties";
+constexpr std::string_view kUpdateArg4 = "removedEdgeProperties";
+constexpr std::string_view kUpdateArg5 = "setVertexLabels";
+constexpr std::string_view kUpdateArg6 = "removedVertexLabels";
+
+/* stats constants */
+constexpr std::string_view kProcedureStatsOnline = "stats_online";
+constexpr std::string_view kProcedureStatsOffline = "stats_offline";
```

Comment:
In MAGE, the `_online` and `_offline` procedures are named `set` and `get` -> in this case `stats_set` and `stats_get`.